Makefile

```diff
@@ -40,8 +40,9 @@ $(BINARY):
 	CGO_ENABLED=0 $(GO) build -a -installsuffix cgo -ldflags="-X main.VERSION=${VERSION}" -o $@ github.com/oauth2-proxy/oauth2-proxy/v7
 
 DOCKER_BUILD_PLATFORM ?= linux/amd64,linux/arm64,linux/ppc64le,linux/arm/v6
+DOCKER_BUILD_RUNTIME_IMAGE ?= alpine:3.15
```

Comment:
I suspect we will forget to override this when we update the dockerfile, are we able to leave it empty?

Reply:
Sadly no, as it makes the `RUNTIME_IMAGE` build-arg empty causing this error
```
error: failed to solve: base name (${RUNTIME_IMAGE}) should not be blank
```